users/forms.py

```diff
@@ -9,3 +10,24 @@ class UserRegistrationForm(UserCreationForm):
     class Meta:
         model = User
         fields: ClassVar[list[str]] = ["nickname", "email", "password1", "password2"]
+
+
+class UserProfileForm(forms.ModelForm):

```

Comment:
Would be good to restrict avatar file size  file size, let's say 2MB.

Reply:
Still, would be good to limit this uploaded file size. I am pretty sure there is a settings in setting.py we could use. 
I'd like to avoid out of memory issues because of sending huge files.

Reply:
done